utils/locale.ts

```diff
@@ -0,0 +1,5 @@
+import { I18N_LOCALES } from '~/i18n/i18n.config'
+
+export function isAvailableI18nLocale(locale: string): locale is typeof I18N_LOCALES[number] {
+    return (I18N_LOCALES as unknown as string[]).includes(locale)
+}
```

Comment:
We should only rely on Nuxt I18n at runtime, not on constants. All the available translations are available on `$i18n.locales`